src/styles/index.css

```diff
@@ -0,0 +1,210 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, Arial, sans-serif;
+  margin: 0;
+}
+
+.header {
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.main {
+  display: flex;
+  justify-content: center;
+}
+
+.is-active {
+  margin: 0;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  display: block;
+  width: 100%;
+  height: 4px;
+  border-radius: 2px;
+  bottom: 0;
+  background-color: #00acdc;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 50px;
+}
+
+.nav__list {
+  margin: 0;
+  display: flex;
+  align-items: center;
+  align-content: center;
+  list-style: none;
+}
+
+.nav__links {
+  margin-left: 20px;
+  position: relative;
+  text-decoration: none;
+  text-align: center;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
+  line-height: 60px;
+}
+
+.nav__content {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav .is-active,
+.nav__links:hover {
+  color: #00acdc;
+}
+
+.card {
+  width: 200px;
+  padding: 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.card__image {
+  display: flex;
+  margin: 16px auto 0;
+  height: 134px;
+  justify-self: center;
+}
+
+.card__name {
+  margin-top: 40px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  text-align: left;
+  color: #060b35;
+}
+
+.card__code {
+  margin-top: 4px;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: left;
+  color: #616070;
+}
+
+.card__price {
+  margin-top: 24px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__text {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__cost {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__button {
+  height: 40px;
+}
+
+.card__buttonContent {
+  margin-top: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+  text-decoration: none;
+  border-radius: 5px;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  color: #fff;
+  background-color: #00acdc;
+  border: 1px solid #00acdc;
+}
+
+.card__buttonContent:hover {
+  color: #00acdc;
+  background-color: #fff;
+}
+
+.stars {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars__stars {
+  display: flex;
+  gap: 4px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__stars .stars__star:nth-child(1n) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars__stars .stars__star:nth-child(1n + 5) {
+  background-image: url(../images/star.svg);
+}
+
+.stars__rev {
```

Comment:
There is a typo in the 'display: flex-end;' property. It should be 'display: flex;' and 'justify-content: flex-end;' to align items to the end of the flex container.